content/docs/command-reference/import-url.md

```diff
@@ -50,8 +50,8 @@ contains the corresponding local path in the <abbr>workspace</abbr>. It records
 enough metadata about the imported data to enable DVC efficiently determining
 whether the local copy is out of date.
 
-Note that `dvc repro` doesn't check or update import `.dvc` files, use
-`dvc update` on them to bring the import up to date from the external data
+Note that `dvc repro` doesn't check or update import `.dvc` files (see
+`dvc freeze`), use `dvc update` to bring the import up to date from the data
```

Comment:
no need to reference `dvc freeze` here I think - it complicates this

Reply:
OK, removed freeze ref.